tfimm/train/__init__.py

```diff
@@ -16,7 +16,17 @@
     SavedModel,
     SavedModelConfig,
 )
-from .optimizer import OptimizerConfig, OptimizerFactory  # noqa: F401
+from .optimizers.optimizer import OptimizerConfig, OptimizerFactory  # noqa: F401
+from .optimizers.lr_schedules import (
+    LrConstConfig,
+    ConstSchedule,
+    LrMultiStepsConfig,
+    MultiStepsSchedule,
+    LrCosineDecayConfig,
+    CosineDecaySchedule,
+    LrExpDecayConfig,
+    ExponentialDecaySchedule,
```

Comment:
Regarding naming, can I suggest synchronising the names of the config class and the schedule, i.e.,
- `LRConstConfig`, `LRConstFactory`
- `LRMultiStepsConfig`, `LRMultiStepsFactory`
- `LRCosineDecayConfig`, `LRCosineDecayFactory`
- `LRExpDecayConfig`, `LRExpDecayFactory`
The reason for calling the Factory rather than Schedule is the same as for `OptimizerFactory`: they are classes to create LR schedules.